### PR TITLE
fix(errors): provide stacktrace with errors

### PR DESCRIPTION
**Pull request by SteveBunlon**
### General

- [x] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Test manually the implemented changes
- [x] Validate the code quality (indentation, syntax, style, simplicity, readability)

### Security

- [x] Consider the security impact of the changes made

### CODE REVIEW

File: config/initializers/errors.rb [hunk 1]
```diff
@@ -42,13 +42,15 @@ def initialize(error, error_description, state)
...
+      attr_reader :error_code, :status, :message, :name, :backtrace
 
-      def initialize(error_code, status, message, name = nil)
+      def initialize(error_code, status, message, name = nil, backtrace = nil)
         @error_code = error_code
         @status = status
         @message = message
         @name = name
+
+        set_backtrace(backtrace) unless backtrace.nil?
```
[the remark this answers is not in the document]
Comment (nicolasalexandre9) [reply config/initializers/errors.rb:53]: Or if you use `set_backtrace` then the `attr_reader :backtrace` is useless

**Review by matthv — COMMENTED** [review]
It looks like two unrelated files were included in this pull request. 

Additionally, it would be great to add some tests to ensure the functionality modified 🙏

File: config/initializers/errors.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (matthv) [reply config/initializers/errors.rb:53]: I like the 2nd option

File: .forestadmin-schema.json
[the remark this answers is not in the document]
Comment (SteveBunlon) [reply .forestadmin-schema.json:1]: remove 👍

File: lib/.DS_Store
[the remark this answers is not in the document]
Comment (SteveBunlon) [reply lib/.DS_Store:1]: removed 👍

**Review by matthv — APPROVED** [review]

**Pull request merged**